cli/src/main/java/com/devonfw/tools/ide/tool/gradle/Gradle.java

```diff
@@ -12,6 +14,16 @@
  */
 public class Gradle extends LocalToolCommandlet {
 
+  /**
+   * The name of the gradle configuration folder.
+   */
+  public static final String GRADLE_CONFIG_FOLDER = "gradle";
+
+  /**
+   * The name of the legacy gradle configuration folder.
+   */
+  public static final String GRADLE_CONFIG_LEGACY_FOLDER = ".gradle";
```

Comment:
Why should we support legacy compatibility for something that never existed?
```suggestion
```

Reply:
Seems not to be used anywhere so surely a leftover so you can just click on `Commit suggestion`. 😄